packages/core/src/components/hotkeys/hotkeysTarget.tsx

```diff
@@ -37,12 +43,19 @@ export function HotkeysTarget<T extends { prototype: IHotkeysTarget }>(construct
         this.localHotkeysEvents = new HotkeysEvents(HotkeyScope.LOCAL);
         this.globalHotkeysEvents = new HotkeysEvents(HotkeyScope.GLOBAL);
 
+        if (componentWillMount != null) {
+            componentWillMount.call(this);
+        }
+    };
+
+    // tslint:disable no-invalid-this only-arrow-functions
```

Comment:
you don't need this line; L41 disables these rules for the rest of the file until a `tslint:enable` is encountered.